docs/Software Architecture Document/History/index.md

```diff
@@ -31,6 +31,8 @@
 | 3.2         | 08/10/2016 | Draft           | [Sergey Levchenko]     | Add AppLaunch Component |
 | 3.3         | 08/11/2016 | Draft           | [Elisey Zamakhov]      | Use cases moved to a separate chapter <br> Add applications data storing in Process State View |
 | 3.4         | 08/16/2016 | Draft           | [Elisey Zamakhov]      | Update Diagnostics chapter with log4cxx features usage |
+| 3.5         | 09/09/2016 | Draft           | [Elisey Zamakhov]      | Change Coding style |
+| 3.6         | 21/09/2016 | Draft           | [Elisey Zamakhov]      | Update BT BlueZ limitation |
```

Comment:
This line seems to be following a different date format from the rest of the document